docs/client-rpc.md

```diff
@@ -35,7 +35,7 @@ None
 
 #### Response
 
-- network: String - [MainNet, Testnet3, RegTest, Signet]
+- network: String - [Mainnet, Testnet, Regtest, Signet]
```

Comment:
these are from a library so they can't change, however looks like I did make a typo, `Signet` should be `SigNet` and `Testnet3` should be `TestNet3`